clients/web/src/utilities/EventStream.js

```diff
@@ -64,6 +73,7 @@ prototype.open = function () {
                 stream.trigger('g:error', e);
                 return;
             }
+            window.localStorage.setItem('sseTimestamp', obj._girderTime);
```

Comment:
+1 - I also think there are addons/settings for various browsers to explicitly limit/disable local storage on a per site basis.